### PR TITLE
Provide the ability to deploy extra objects

### LINKED ISSUE #41: Add the ability to deploy extra objects
`.Values.enableStartupScripts` allows to mount ConfigMap to `/docker-entrypoint-initaws.d`, but there is no way to create this ConfigMap. To resolve this problem I suggest to add the ability to deploy extra objects like it done for almost all Bitnami charts

### PR DESCRIPTION
**Pull request by pstetsuk-exadel**
Fixes #41

### CODE REVIEW

File: charts/localstack/README.md [hunk 1]
```diff
@@ -135,6 +136,7 @@ When raising a pull request with a fix or new feature, please make sure to:
 
 ## Change Log
 
+* v0.3.5: Add the ability to deploy extra objects
```
Comment (cmeans) [inline charts/localstack/README.md:139]: Wrong version...
```suggestion
* v0.3.6: Add the ability to deploy extra objects
```

**Review by cmeans — CHANGES_REQUESTED** [review]
I'd suggest, one minor change.

**Review by cmeans — APPROVED** [review]
LGTM

**Comment by whummer** [comment]
Great, thanks @pstetsuk-exadel ! 👍

**Comment by LaikaN57** [comment]
This seems a bit hammer-fisted at first glance but if this is a bitnami pattern (as stated by op in #41) then I guess this is fine :/

**Pull request merged**